src/main/drivers/sensor.h

```diff
@@ -34,10 +34,13 @@ typedef enum {
     CW270_DEG_FLIP = 8
 } sensor_align_e;
 
-typedef union busDevice_t {
+typedef union busDevice_u {
```

Comment:
@martinbudden : There is problem with mpu detection - you will have to use bus-specific detection routine.